OpenDocumentReader.xcodeproj/project.pbxproj

```diff
@@ -12,6 +12,8 @@
 		AC384BCD23B4FFA700C7BF47 /* ContentViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = AC384BC923B4FFA700C7BF47 /* ContentViewController.swift */; };
 		AC384BCE23B4FFA700C7BF47 /* Constants.swift in Sources */ = {isa = PBXBuildFile; fileRef = AC384BCA23B4FFA700C7BF47 /* Constants.swift */; };
 		AC384BCF23B4FFA700C7BF47 /* PageViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = AC384BCB23B4FFA700C7BF47 /* PageViewController.swift */; };
+		AC6A1CD9243E6B8400CD42EC /* (null) in Sources */ = {isa = PBXBuildFile; };
```

Comment:
what's that? 🤷

Reply:
weird, i recreated both configurations from scratch. here ist the that i used: [guide](https://www.youtube.com/watch?v=r242pYLr-Gg&t=)